lib/errors.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const getContent = (err, debug, translate) => {
+  var content = {};
+
+  if (err.code === 'SESSION_TIMEOUT') {
+    content.title = translate('errors.session.title');
```

Comment:
not sure what i think of this. If feels quite frameworky. We are now saying you want a middleware you must have a `translate`. The idea of the cookie middleware was to keep it very light...